x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/jira/case_mappings/field_mapping.tsx

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
It make sense to move all files, which is common for ServiceNow, Jira and Resilient to the common folder case_mappings to avoid the code duplication